cirq-core/cirq/devices/device.py

```diff
@@ -226,11 +228,14 @@ def nx_graph(self) -> Optional['nx.Graph']:
     def _value_equality_values_(self):
         graph_equality = None
         if self._nx_graph is not None:
-            graph_equality = (sorted(self._nx_graph.nodes()), sorted(self._nx_graph.edges()))
+            graph_equality = (
+                tuple(sorted(self._nx_graph.nodes())),
+                tuple(sorted(self._nx_graph.edges())),
+            )
```

Comment:
Since we are taking an `nx.Graph` as input in this class, it's possible to pass a directed or undirected graph which has the same set of nodes and edges. The two should not compare to be equal, but in this case they would.

Reply:
Oops, added that in along with a test. I could've sworn that the `edges()` function determined whether or not an edge was directed.